Eyes/src/Modules/InitializeEyes.cs

```diff
@@ -42,6 +43,13 @@ public InitializeEyes()
         /// that will in turn invoke this method.</remarks>
         void ITestModule.Run()
         {
+            if (Configuration.Current.Tools.UseUiaLauncher)
+            {
+                Report.Failure("Ranorex.Eyes is not supporting UIA Launcher and disabled it for this solution. Please start this test again.");
+                Configuration.Current.Tools.UseUiaLauncher = false;
+                Configuration.Current.SaveToUserSettings();

```

Comment:
This will most probably disable the UiaLauncher system wide, not just for that solution, since invoked at runtime -> update error message?